models/repo_mirror.go

```diff
@@ -260,10 +262,23 @@ func SyncMirrors() {
 		}
 
 		m.ScheduleNextUpdate()
-		if err = UpdateMirror(m); err != nil {
+		if err = updateMirror(sess, m); err != nil {
 			log.Error(4, "UpdateMirror [%s]: %v", repoID, err)
 			continue
 		}
+
+		// Get latest commit date and compare to current repository updated time,
+		// update if latest commit date is newer.
+		commitDate, err := git.GetLatestCommitTime(m.Repo.RepoPath())
```

Comment:
Comment should also be fixed as there is no check anymore

Reply:
See https://github.com/go-gitea/gitea/pull/3551/commits/ac27f460a0b43f4daad960975b5b71718a0fa9be